### PR TITLE
[MDC] Update documentation to reflect how binding/unbinding services work.

### PR DESCRIPTION
**Pull request by wojta**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-7934

## What
Updated documentation for MDC changes in binding/unbinding services.

## How
Changed section `Mobile Developer Console > Configuring Mobile Services`

## Checklist:
- [x] Updated docs
- [ ] Changes have been successfully verified by another team member 

## Progress
- [x] Finished task

## Additional Notes
Screenshots still need to be updated in future.

### CODE REVIEW

**Comment by finp** [comment]
@wojta  is there an environment we can try out MDC binding? cc @jstaffor

**Comment by jhellar** [comment]
@finp it should be possible to test it on https://comm2.skunkhenry.com:8443

**Comment by wojta** [comment]
@finp There are unfortunately still some bugs in this functionality, but from my understanding it should work as I wrote it in the documentations.

**Comment by laurafitzgerald** [comment]
@wojta when do we expect we'll have screenshots for this? If they need to be updated after the fact can you go ahead and create a JIRA issue under the epic for that and link it as blocked by the epic tracking the work for binding/unbinding. @finp am I right in saying we should aim to keep images to a minimum for this reason?

**Comment by laurafitzgerald** [comment]
@wojta @finp I'm getting the following error when running `./bin/build.sh`. is anyone else seeing that?
```
sed: 1: "./build/site/sitemap.xml": invalid command code .
```

**Comment by finp** [comment]
@laurafitzgerald fixed with https://github.com/aerogear/mobile-docs/pull/529
(hopefully)

**Comment by laurafitzgerald** [comment]
@wojta mdc-docs branch is now rebased on master (thanks @jhellar can you rebase this pr against mdc-docs in order to pull in the fix.

**Comment by laurafitzgerald** [comment]
@wojta what is the status of this pr. Does it still need review or can it be merged?

File: modules/ROOT/pages/_partials/mobile-client-configuring-mobile-services.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+At the second step you can fill in configuration details for the binding (image is the example of a Push Service). 
+
+image::mobile-clients-service-bind-2.png[]
+
+Press *Create* to create the binding (image is the example of a Push Service).
+
+image::mobile-clients-service-bind-3.png[]
+
+Once final step is finished the service is bound and the configuration in `mobile-services.json` for the {mobile-client} will be updated in the
```
Comment (finp) [inline modules/ROOT/pages/_partials/mobile-client-configuring-mobile-services.adoc]: After the binding is complete, the configuration in .... is updated ..

**Comment by wojta** [comment]
updating now

File: modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
+= Using mobile services
+
+== Viewing mobile services
+
+In order to use services with a mobile app, the {mobile-client} associated with the app must be bound to services. 
+To show the list of services, go to the application *Overview* screen and to the *mobile services* tab.
```
Comment (aliok) [inline modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc]: *mobile services* --> *Mobile Services*

File: modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+In order to use services with a mobile app, the {mobile-client} associated with the app must be bound to services. 
+To show the list of services, go to the application *Overview* screen and to the *mobile services* tab. 
+There is a list of services available to be bound to the {mobile-client} and list of services that have already been bound to the client.
+
+image::mobile-clients-mobile-services.png[]
+
+== Binding mobile services
+
+The *Mobile Services* tab in the {mobile-client} view shows the provisioned services that are available to be bound
+to the {mobile-client}. If a service has not yet been bound to a {mobile-client}, a binding can be created by selecting
```
Comment (aliok) [inline modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc]: by selecting --> by clicking?

File: modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+the *Create Binding* button.
+
+You can bind a service by clicking *Bind to app* (image is the example of a Push Service).
```
Comment (aliok) [inline modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc]: *Bind to app* --> *Bind to App*

Comment (wojta) [reply modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc]: ok

**Comment by wojta** [comment]
@finp Can you make final review?

**Pull request merged**